File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/UserRole.java
package com.drivingschoolrwandaapp.models.entities;
import com.google.gson.annotations.SerializedName;
import java.io.Serializable;
import java.util.List;

public class UserRole implements Serializable {
    @SerializedName("id")
    private int id;

    @SerializedName("roleName")
    private String roleName;

    @SerializedName("description")
    private String description;

    // Getters and Setters
    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
    public String getRoleName() { return roleName; }
    public void setRoleName(String roleName) { this.roleName = roleName; }
    public String getDescription() { return description; }
    public void setDescription(String description) { this.description = description; }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/UserRole.kt
package com.drivingschoolrwandaapp.models.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class UserRole(
    @SerializedName("id") var id: Int = 0,
    @SerializedName("roleName") var roleName: String? = null,
    @SerializedName("description") var description: String? = null
) : Serializable
